.travis.yml

```diff
@@ -47,47 +47,76 @@ env:
     - CCACHE_SLOPPINESS=time_macros,file_macro
     - CCACHE_BASEDIR="$TRAVIS_BUILD_DIR"
 
+anchors:
+  envs: &common-job-env
+    # Do not fall back to cache for "master" for PR on "release" branch:
+    # adds the target branch to the cache key.
+    __FOR_TRAVIS_CACHE__=$TRAVIS_BRANCH
```

Comment:
Results in the following in job details - not really nice..

> CI_TARGET=lint __FOR_TRAVIS_CACHE__=$TRAVIS_BRANCH

Should we use something shorter?

> CI_TARGET=lint TCK=$TRAVIS_BRANCH

Or just remove the "__" prefix/suffix?

> CI_TARGET=lint FOR_TRAVIS_CACHE=$TRAVIS_BRANCH

Reply:
why not use `name:` as in the `release-0.3` PR?

Reply:
Yeah, only relevant for the job detail view.